src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('.tree li').forEach((li) => {
+  const childNodes = Array.from(li.childNodes);
```

Comment:
Using `Array.from()` is correct here to convert `childNodes` to an array, allowing the use of array methods. Ensure that this is necessary for your use case, as it may add overhead.